readyapi4j-core/src/test/java/com/smartbear/readyapi4j/RestRequestStepRecipeTest.java

```diff
@@ -394,6 +394,7 @@ public void buildRestRequestTestRecipeWithPropertyExtractor() {
         final String[] extractedProperty = {""};
         TestRecipe recipe = newTestRecipe(
                     POST(URI)
+                            .named("RestRequest")
                         .withExtractors(
                                 propertyExtractor("Endpoint", property -> extractedProperty[0] = property)))
```

Comment:
I'm still not entirely happy with the naming of these methods - perhaps we could rename this method to "fromProperty" - and the below method to "fromResponse" ?